### DISCUSSION TITLE
Use of List Columns vs Site Column with PnP Search v3

### DISCUSSION THREAD

**Discussion by hsamdani12** [microsoft-search/pnp-modern-search#2508] [Q&A]
Hi,  I'm using PnP Search v3, and I was able to map regular list column - crawled properties to pre-defined Refinable managed properties. And then was able to use them successfully in the PnP Search Result and Refiner webparts. 

Has it changed v4 to only work with Site Column and not work List Columns ?  
I was thinking of replacing the v3 with v4, but that means I would need to create all the site columns and then copy over values from the list columns and then delete the list columns ?
As I see the following statement -
"In order to use your custom column values as filters in your PnP Modern Search search solutions, those columns must be Site Columns (usually included in Content Types) and they must be mapped to Managed Properties."

**Comment by wobba** [comment]
v3 and v4 are equal in terms of functionality regarding API usage. Can you point out where in the docs this is written? Site columns only refer to translation and use of taxonomy columns.

**Comment by hsamdani12** [comment]
thanks. In v4.0 documentation, it clearly says it needs to be site columns(usually included in Content Type) - under set up managed properties.
https://microsoft-search.github.io/pnp-modern-search/scenarios/ <https://microsoft-search.github.io/pnp-modern-search/scenarios/>

If it supports list columns which can then be manually(crawled properties) mapped to the in-build refiner managed properties, then it should not say this.
As I mentioned I was able to use the list columns in v3.0.

**Reply by wobba** [reply]
This is incorrect and I have amended the scenario documentation in the dev branch. We'll push it out later. For a filter you need a refinable managed property, so no change.

**Reply by hsamdani12** [reply]
@wobba  ok thanks. Yes I did have them as refinable - using the preexisting ones.
